website/content/configuration/_advanced_ipaddresspool_configuration.md

```diff
@@ -109,3 +109,21 @@ misguided
 If you encounter this issue with your users or networks, you can
 set the `AvoidBuggyIPs` flag of the IPAddressPool CR.
 By doing so, the `.0` and the `.255` addresses will be avoided.
+
+### Changing the IP of a service
+
+The current behaviour of MetalLB is to try to preserve the connectivity when a change of
+configuration that might disrupt a service happens. This includes removing IPs assigned to services.
```

Comment:
this reads a bit like the opposite (MetalLB does remove..) wdyt about:
`when a change of` -> `despite a change of`
`this includes` -> `e.g one that removes..` ?